### DISCUSSION TITLE
Z offset too low.

### DISCUSSION THREAD

**Discussion by PaulStep61** [OrcaSlicer/OrcaSlicer#1206] [Q&A]
Greetings, 
I decided to give Orca a whirl. 
I've been using S3D for a couple of years but need to have the ability to use newer capabilities (adaptive bed mesh, exclude object, etc.).

So I have no issues printing on my V2.4  with files sliced in S3D,
I tried Orca today and the nozzle is way too low, so low that the extruder could not extrude. The filament jammed at the end of the nozzle and was deformed (mushroomed) and could not be pulled out of the hot end.

Via the Device tab in Orca - When I do a Probe Calibrate with paper and accept the new height it is coming out at 0.9, this is 0.3 higher the value reached when accessing the printer via Chrome.

Is there a hidden setting where the Z offset is offset?

The deformed filament.
![20230531_114228](https://github.com/SoftFever/OrcaSlicer/assets/116076473/ee140dba-db21-4025-9c80-47738f81dd10)

**Comment by jrocketfingers** [comment]
Have you figured this out yet? I'm experiencing the same thing, while having successful prints with super/prusa slicers.

**Reply by PaulStep61** [reply]
No not yet. I'm in the middle of a major upgrade (EBB SB2209 & CNC TAP) so I've had no time to do any more research.

**Comment by Ruiaxe** [comment]
Hi!

I also have this issue.

Cheers,
Rui

**Comment by zerquetch** [comment]
Geez, getting on 3 yrs. Did you have enough time to figure out an answer yet?
